### ISSUE TITLE
generator + computed property method is parsed as none computed.

### ISSUE THREAD

**Issue by h13i32maru** [eslint/espree#207]
Hi! ESLint Team. I found the following problem. Is this the correct spec?

- If a method is generator and computed property, the AST node has ``computed: false``.

```javascript
export default class A {
  *[foo]() {}
}
```

```javascript
            {
              "type": "MethodDefinition",
              "key": {
                "type": "Identifier",
                "name": "foo",
                "range": [
                  29,
                  32
                ]
              },
              "value": {
                "type": "FunctionExpression",
                "id": null,
                "params": [],
                "body": {
                  "type": "BlockStatement",
                  "body": [],
                  "range": [
                    36,
                    38
                  ]
                },
                "generator": true,
                "expression": false,
                "range": [
                  33,
                  38
                ]
              },
              "kind": "method",
              "computed": false, // <= I expect ``true``
              "range": [
                27,
                38
              ],
              "static": false
            }
```

http://felix-kling.de/esprima_ast_explorer/#/FZHNajeXmq

**Comment by nzakas** [comment]
That looks like a bug. It should be fixed when we switch to Acorn.

**Comment by nzakas** [comment]
Working on this.

**Comment by nzakas** [comment]
Verified this is working on master and a test exists for it.
